composer/trainer/trainer.py

```diff
@@ -1130,7 +1130,7 @@ def __init__(
                     'Specifying a `file_path_format_string` to a `RemoteUploaderDownloader` is not currently supported while using `save_latest_filename`. '
                     'Please specify the path formatting via `save_folder`, `save_filename`, and `save_latest_filename`')
 
-        # Callbacks
+        # Callbacks.
```

Comment:
```suggestion
        # Callbacks
```
idt we use periods anywhere else?

Reply:
sorry that was just to trigger jenkins again when it was taking too long. forgot to remove it.